recipe/meta.yaml

```diff
@@ -12,16 +16,19 @@ source:
   fn: cfe-{{ version }}.src.tar.xz
   url: http://llvm.org/releases/{{ version }}/cfe-{{ version }}.src.tar.xz
   sha256: {{ sha256 }}
-{% else %}
+{% elif clang_variant.startswith("cling") %}
+  git_url: http://root.cern.ch/git/clang.git
+  git_rev: {{ clang_variant|replace('cling_', 'cling-') }}
+{% elif clang_variant.startswith("root") %}
   git_url: https://github.com/root-project/root.git
-  git_rev: {{ clang_variant|replace('cling_', 'v')|replace('.', '-') }}
+  git_rev: {{ clang_variant|replace('root_', '')|replace('.', '-') }}
   folder: root-source
 {% endif %}
 
 build:
   number: {{ build_number }}
-  skip: true  # [win and vc<14]
-
+  # skip: true  # [win and vc<14]
+  skip: true  # [win]
```

Comment:
Was this intentional? I guess it was done for the rerendering bug? If so, I previously worked around it by removing this line, deleting the vc9 build config then re-adding this line.

Reply:
ok, will try that.